static/styles/image_upload_widget.scss

```diff
@@ -184,6 +185,12 @@
         top: 30%;
         right: 30%;
     }
+
+    .image_file_input_error {
+        height: 30px;
+        width: 200px;
+        display: block;
+    }
```

Comment:
Can you post before/after screenshots for these prep commits?

Reply:
ok sure